man/repart.d.xml

```diff
@@ -734,6 +734,38 @@
 
         <xi:include href="version-info.xml" xpointer="v253"/></listitem>
       </varlistentry>
+
+      <varlistentry>
+        <term><varname>MountPoint=</varname></term>
+
+        <listitem><para>Specifies where and how the partition should be mounted. Takes at least one and at
```

Comment:
this should have some reference to fstab(5) man page at some good place

Reply:
and should probably say that this corresponds with the 2nd and 4th column of fstab